documentation/library_guide/introduction.rst

```diff
@@ -138,6 +138,9 @@ Known Limitations
   the dereferenced value type of the provided iterators should satisfy the ``DefaultConstructible`` requirements.
 * For ``remove``, ``remove_if``, ``unique`` the dereferenced value type of the provided
   iterators should be ``MoveConstructible``.
+* The algorithms which process uninitialized storage: ``uninitialized_copy``, ``uninitialized_copy_n``, ``uninitialized_fill``, ``uninitialized_fill_n``, ``uninitialized_fill_n``, ``uninitialized_move``, ``uninitialized_move_n``, ``uninitialized_default_construct``, ``uninitialized_default_construct_n``, ``uninitialized_value_construct``, ``uninitialized_value_construct_n``
```

Comment:
My suggestion:

* The algorithms that process uninitialized storage: ``uninitialized_copy``, ``uninitialized_copy_n``, ``uninitialized_fill``, ``uninitialized_fill_n``, ``uninitialized_fill_n``, ``uninitialized_move``, ``uninitialized_move_n``, ``uninitialized_default_construct``, ``uninitialized_default_construct_n``, ``uninitialized_value_construct``, ``uninitialized_value_construct_n`` should be called with a device policy when using device data and should be called with a host policy when using host data.  Otherwise, the result is undefined.
* The algorithms that destroy data: ``destroy`` and ``destroy_n`` should be called with a host policy when using host data initialized on the host and should be called with a device policy when using device data initialized on the device. Otherwise, the result is undefined.

Reply:
My suggestions in **bold**:
* The algorithms **that** process uninitialized storage: uninitialized_copy, uninitialized_copy_n, uninitialized_fill, uninitialized_fill_n, uninitialized_fill_n, uninitialized_move, uninitialized_move_n, uninitialized_default_construct, uninitialized_default_construct_n, uninitialized_value_construct, uninitialized_value_construct_n should be called with a device policy when using device data and should be called with a host policy when using host data. Otherwise, the result is undefined. 
* The algorithms **that** destroy data: destroy, destroy_n, should be called with a host policy when using host data **that** was initialized on the host, and should be called with a device policy when using device data **that** was initialized on the device. Otherwise, the result is undefined.

A note: make sure the bullet points render correctly. Currently, the last two bullets are in the same entry.